mappings/net/minecraft/text/MutableText.mapping

```diff
@@ -0,0 +1,37 @@
+CLASS net/minecraft/class_5250 net/minecraft/text/MutableText
+	COMMENT A text with mutation operations.
+	COMMENT
+	COMMENT @see Text
```

Comment:
```suggestion
```

Not needed as it's linked right next to this comment in both javadoc and the game code.

Reply:
too lazy to remove